lib/cli/src/commands/publish.rs

```diff
@@ -44,17 +52,134 @@ pub struct Publish {
     /// for each individual query to the registry during the publish flow.
     #[clap(long, default_value = "2m")]
     pub timeout: humantime::Duration,
+
+    /// Whether or not the patch field of the version of the package - if any - should be bumped.
+    #[clap(long)]
+    pub autobump: bool,
+
+    /// Do not prompt for user input.
+    #[clap(long)]
+    pub non_interactive: bool,
```

Comment:
This should fall back to a default value

Reply:
[62f0c2c](https://github.com/wasmerio/wasmer/pull/4532/commits/62f0c2c7bfa19edcd7980324e52643ed78c6513a)